packages/okta-react/package.json

```diff
@@ -12,7 +12,8 @@
     "prestart": "npm run build && npm run build:harness",
     "start": "npm run --prefix test/e2e/harness start",
     "pretest": "npm run build && npm run build:harness",
-    "test": "npm run --prefix test/e2e/harness test"
+    "test": "npm run --prefix test/e2e/harness test",
+    "install:package": "npm install"
```

Comment:
It this `install:package` needed?

Reply:
😕

Reply:
This was related to some tests having local node_modules dependencies.
Since we `hoist = true` in `lerna.json`, these dependencies would be installed in the root `node_modules` which would cause these tests to fail.
As discussed with @robertjd, I'm removing `hoist = true` after which we can remove `install:package`